Final_project_2/frontend/css/style.css

```diff
@@ -517,21 +520,14 @@ address .fa {
   
 }
 
-textarea.form-control {
+textarea.contact-form-input {
   height: 100px;
 }
 
 .btn-send {
-  margin-left: 50%;
+  float: right;
```

Comment:
this makes it now aligned with the textarea above 🎉 

Note that this change alone wouldn't have worked - it depends on the rest of the form being fixed to have proper sizing etc.